Forge.TreeWalker/src/ForgeSchemaValidator.cs

```diff
@@ -0,0 +1,284 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ForgeSchemaValidator.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// <summary>
+//     The ForgeSchemaValidator class implements the ITreeSchemaValidator interface.
+// </summary>
+//-----------------------------------------------------------------------
+namespace Microsoft.Forge.TreeWalker
```

Comment:
nit: Add newline between the copyright section and namespace.

Also add a newline between the using statements and the class summary section.